horace_core/sqw/@sqw/sigvar_set.m

```diff
@@ -6,12 +6,9 @@
 
 w = copy(win);
 w.data = sigvar_set(win.data,sigvar_obj);
-
```

Comment:
Given this is a method on `sqw` do we need this check?

Reply:
well, this is `sqw` method and DnD method. `sqw`  `DnD` method works first and if it is `sqw`,  it does a bit more. 
so yes, the check is necessary.

Reply:
This is only an `sqw` method in `sqw/@sqw` DnD shouldn't be able to see this.

Reply:
yes it does not but pixelless sqw object may come as input.